package.json

```diff
@@ -8,7 +8,7 @@
   },
   "type": "module",
   "scripts": {
-    "dev": "next dev --port=6060",
+    "dev": "npm run build-nimbus && next dev --port=6060",
```

Comment:
I could also run `npm run build-glean` in here, but I think that might take a lot longer the first time? On the other hand, you need to do it anyway I think?